backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/HasNvramDataQuery.java

```diff
@@ -17,7 +18,7 @@ public HasNvramDataQuery(P parameters, EngineContext engineContext) {
 
     @Override
     protected void executeQueryCommand() {
-        Pair<String, String> result = vmDao.getNvramData(getParameters().getId());
+        Pair<SecretValue<String>, String> result = vmDao.getNvramData(getParameters().getId());
         getQueryReturnValue().setReturnValue(result.getFirst() != null);
```

Comment:
Looking into VmDaoImpl.getNvramData() I don't see any case when it may return null. But the value may be `null`.

Reply:
Done.